frontend/components/about/Values.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+
+import { values } from './index';
+
+</script>
+
+<template> 
+    <ul class="values__list">
+        <AboutValuesItems
+            v-bind = "item"
+            v-for = "item in values"
```

Comment:
Рекомендации линтера:
- Пропс `v-for` должен идти перед `v-bind`;
- Пробелы вокруг знака `=` в пропсах не нужны;
```suggestion
            v-for="item in values"
            v-bind="item"
```